flocker/node/_docker.py

```diff
@@ -534,6 +534,8 @@ def _list():
                     # have been removed in another thread.
                     if e.response.status_code == NOT_FOUND:
                         continue
+                    else:
+                        raise
```

Comment:
Actually, I guess that's probably a consequence of having the majority of the docker functional tests fail early with that `Env` error.  I'll come back to this after that `Env` branch is merged.